### PR TITLE
add new mutation to set the active to

### PR DESCRIPTION
**Pull request by meletios**

### CODE REVIEW

File: src/schema.graphqls [hunk 1]
```diff
@@ -31,6 +31,7 @@ type Mutation {
   editLastResponse: Boolean!
   updateEmail(input: String!): Member!
   updatePhoneNumber(input: String!): Member!
+  activateAtDate(id: UUID!, activationDate: String! ): Boolean!
```
Comment (Iamsamwhoami) [inline src/schema.graphqls]: Shouldn't this be `activateOnDate`? 🤔 https://www.englishclub.com/grammar/prepositions-at-in-on-time.htm

Comment (Iamsamwhoami) [reply src/schema.graphqls]: And I think you can skip `date` in the mutation, that is implied.

Comment (Iamsamwhoami) [reply src/schema.graphqls]: also, maybe rename `id` to `insuranceId` to make it clearer what the ID is for

Comment (Iamsamwhoami) [reply src/schema.graphqls]: And `activationDate` should be a `Date!` not a `String!` I guess

Comment (Iamsamwhoami) [reply src/schema.graphqls]: so `activateOn(date: Date!, insuranceId: UUID!)`

Comment (gustafguner) [reply src/schema.graphqls]: Yeah, you're right. Makes more sense 👍

**Comment by gustafguner** [comment]
This PR is dependent on PR #138 in product-pricing: https://github.com/HedvigInsurance/product-pricing/pull/138. 

Giraffe now uses the new endpoint `/insurance/{memberId}/requestStartDate` instead.

**Pull request closed without merging**